ui/components/app/modals/export-private-key-modal/index.scss

```
@@ -43,17 +43,20 @@
     padding: 10px 0 13px 17px;
     width: 291px;
     height: 44px;
+    background: var(--color-background-default);
+    color: var(--color-text-default);
+    border: 1px solid var(--color-border-default);
   }
 
   &__password::-webkit-input-placeholder {
-    color: var(--dusty-gray);
+    color: var(--color-text-muted);
   }
 
   &__password--warning {
     @include H7;
 
     border-radius: 8px;
-    background-color: #fff6f6;
+    background-color: var(--color-error-inverse);
```

I think you want --color-error-muted here
```suggestion
    background-color: var(--color-error-muted);
```